src/js/views/patients/widgets/widgets.scss

```diff
@@ -7,14 +7,17 @@
 }
 
 .widgets__form-widget {
+  align-items: center;
+  display: flex;
+  max-width: 204px;
+  min-width: 0;
+  padding: 8px 8px 8px 16px;
   width: 100%;
 
   .fa-up-right-and-down-left-from-center {
     font-size: 12px;
-    margin-right: 0;
-    position: absolute;
-    right: 8px;
-    top: 8px;
+    margin-left: auto !important;
```

Comment:
I really don't like using `!important`, but this is being overriden by the `buttons.scss` module styles for this ([here](https://github.com/RoundingWell/care-ops-frontend/blob/develop/src/scss/modules/buttons.scss#L36-L38)).

Maybe there's another way to avoid using `!important`. Not sure though.

Reply:
I think you could wrap the svg in a span to avoid this

Reply:
Yup that removed the need for the `!important`, but had to move some things around since some styles now need to be applied to that `span` instead of the `svg`.